course_discovery/apps/api/v1/tests/test_views/test_course_runs.py

```diff
@@ -279,6 +303,38 @@ def test_contains_multiple_course_runs(self):
             }
         )
 
+    def test_contains_multiple_course_runs_edx_org_short_name_filter(self):
+        """
+        Verify contained course runs filtering on edX organization.
+        """
+        elastic_search_query_string = urllib.parse.urlencode({
+            'query': 'id:course*',
+            'course_run_ids': '{course_run_1_key},{course_run_2_key},{course_run_3_key}'.format(
+                course_run_1_key=self.course_run.key,
+                course_run_2_key=self.course_run_2.key,
+                course_run_3_key='course-v1:edX+DemoX+Demo_Course'
+            )
+        })
+        elastic_search_query_string_with_org_filter = '{elastic_search_query_string}&org={edx_org_short_name}'.format(
+            elastic_search_query_string=elastic_search_query_string,
+            edx_org_short_name=self.edx_org_short_name
+        )
+        course_run_api_url_with_org_filter = '{course_run_api_url}?q={elastic_search_query_string_with_org_filter}'.format(
+            course_run_api_url=reverse('api:v1:course_run-list'),
+            elastic_search_query_string_with_org_filter=elastic_search_query_string_with_org_filter
+        )
+        expected__serialized_contained_course_runs = {
+            'course_runs': {
+                self.course_run_3.key: True,
+                'course-v1:edX+DemoX+Demo_Course': False
```

Comment:
declare a variable `edx_org_course_run_key` for storing this course run key

Reply:
Done